### ISSUE TITLE
False positive `RUF100`

### ISSUE THREAD

**Issue by amyheather** [lintquarto/lintquarto#120]
If use lintquarto to run`ruff` with `RUF` rules enabled, will return `RUF100` as a false positive for the `# noqa` messages add by `lintquarto` - for example:

```
RUF100 [*] Unused `noqa` directive (unused: `E501`; non-enabled: `E302`, `E305`)
 --> docs/pages/tools/examples/typecheck_example.qmd:8:42
  |
6 | # -
7 | # %% [python]
8 | def add_numbers(a: int, b: int) -> int:  # noqa: E302,E305,E501
  |                                          ^^^^^^^^^^^^^^^^^^^^^^
9 |     return a + b
  |
help: Remove unused `noqa` directive
```

Think solution would just be to disable `RUF100` in command e.g., 

```
ruff check . --config "lint.ignore = ['RUF100']"
```

As long as that doesn't override provided config.

And would need to note that this is disabled on `ruff` page of docs, explain that it is not compatible.

**Comment by sneakers-the-rat** [comment]
part of: https://github.com/pyOpenSci/software-submission/issues/257

was going to raise this, i would like to see this change :)
